packages/lib/src/vis/shared/Html.tsx

```diff
@@ -57,25 +60,27 @@ function Html(props: Props) {
   const group = useRef<Group>(null!); // eslint-disable-line @typescript-eslint/no-non-null-assertion
 
   const getGroupPosition = useCallback(() => {
-    const groupPos = v1.setFromMatrixPosition(group.current.matrixWorld);
-    groupPos.project(camera);
+    const rawGroupPos = v1.setFromMatrixPosition(group.current.matrixWorld);
```

Comment:
I think you can initialize a new vector on the fly here (and below as well with `v1` again and `sizeVector`). Also, maybe rename the vars to `cameraPos` and `htmlPos` similarly to what you did for the scale in `TooltipMesh`?

```ts
const worldPos = new Vector3().setFromMatrixPosition(group.current.matrixWorld);
const cameraPos = worldPos.clone().project(camera);
const htmlPos = projectCameraToHtml(cameraPos , width, height);
```

You could also chain the call to `project` to avoid the extra `worldPos` constant initialization and cloning, but as discussed I reckon the performance impact is not significant, and two variables makes the code more readable in my opinion.